repository/spark/docs/3.0.0-1.1.0/versions.md

```diff
@@ -0,0 +1,10 @@
+# Versions
+
+## Component Versions
+* Apache Spark 3.0.0 built with Hadoop 2.9.2 and Scala 2.12
+* Spark Operator v1beta2-1.1.1
+* OpenJDK 8
+* Prometheus Java agent 0.11.0
+
+## KUDO Version Requirement
+* KUDO version 0.13.0 or later
```

Comment:
Please, cross-check which KUDO version supports K8s 1.19. IIRC it is _at least_ 0.15.0 but maybe newer. Needs to be updated here and everywhere.
```suggestion
* KUDO version 0.15.0 or later
```